[2022-12-28 17:16:37,209] {processor.py:153} INFO - Started process (PID=10141) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:16:37,210] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:16:37,216] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,216] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:16:37,274] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:16:37,442] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,441] {manager.py:508} INFO - Created Permission View: can edit on DAG:dados_climaticos
[2022-12-28 17:16:37,468] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,467] {manager.py:508} INFO - Created Permission View: can read on DAG:dados_climaticos
[2022-12-28 17:16:37,489] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,489] {manager.py:508} INFO - Created Permission View: can delete on DAG:dados_climaticos
[2022-12-28 17:16:37,489] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,489] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:16:37,511] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,511] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 17:16:37,562] {logging_mixin.py:115} INFO - [2022-12-28 17:16:37,562] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:16:37,613] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.408 seconds
[2022-12-28 17:17:09,263] {processor.py:153} INFO - Started process (PID=10195) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:17:09,263] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:17:09,265] {logging_mixin.py:115} INFO - [2022-12-28 17:17:09,264] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:17:09,277] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:17:09,347] {logging_mixin.py:115} INFO - [2022-12-28 17:17:09,347] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:17:09,375] {logging_mixin.py:115} INFO - [2022-12-28 17:17:09,374] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:17:09,427] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.169 seconds
[2022-12-28 17:17:47,302] {processor.py:153} INFO - Started process (PID=10277) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:17:47,303] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:17:47,304] {logging_mixin.py:115} INFO - [2022-12-28 17:17:47,304] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:17:47,317] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:17:47,389] {logging_mixin.py:115} INFO - [2022-12-28 17:17:47,389] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:17:47,426] {logging_mixin.py:115} INFO - [2022-12-28 17:17:47,426] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:17:47,456] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.159 seconds
[2022-12-28 17:18:24,411] {processor.py:153} INFO - Started process (PID=10355) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:18:24,413] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:18:24,414] {logging_mixin.py:115} INFO - [2022-12-28 17:18:24,414] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:18:24,426] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:18:24,454] {logging_mixin.py:115} INFO - [2022-12-28 17:18:24,453] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:18:24,484] {logging_mixin.py:115} INFO - [2022-12-28 17:18:24,484] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:18:24,511] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.104 seconds
[2022-12-28 17:19:03,634] {processor.py:153} INFO - Started process (PID=10435) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:19:03,639] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:19:03,640] {logging_mixin.py:115} INFO - [2022-12-28 17:19:03,640] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:19:03,652] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:19:03,679] {logging_mixin.py:115} INFO - [2022-12-28 17:19:03,679] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:19:03,710] {logging_mixin.py:115} INFO - [2022-12-28 17:19:03,710] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:19:03,736] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.107 seconds
[2022-12-28 17:19:35,138] {processor.py:153} INFO - Started process (PID=10496) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:19:35,139] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:19:35,140] {logging_mixin.py:115} INFO - [2022-12-28 17:19:35,139] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:19:35,151] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:19:35,178] {logging_mixin.py:115} INFO - [2022-12-28 17:19:35,178] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:19:35,210] {logging_mixin.py:115} INFO - [2022-12-28 17:19:35,210] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:19:35,236] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.103 seconds
[2022-12-28 17:20:08,319] {processor.py:153} INFO - Started process (PID=10555) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:20:08,321] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:20:08,322] {logging_mixin.py:115} INFO - [2022-12-28 17:20:08,322] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:20:08,338] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:20:08,381] {logging_mixin.py:115} INFO - [2022-12-28 17:20:08,381] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:20:08,419] {logging_mixin.py:115} INFO - [2022-12-28 17:20:08,418] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:20:08,447] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.135 seconds
[2022-12-28 17:20:50,429] {processor.py:153} INFO - Started process (PID=10644) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:20:50,433] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:20:50,434] {logging_mixin.py:115} INFO - [2022-12-28 17:20:50,434] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:20:50,448] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:20:50,523] {logging_mixin.py:115} INFO - [2022-12-28 17:20:50,523] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:20:50,563] {logging_mixin.py:115} INFO - [2022-12-28 17:20:50,563] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:20:50,593] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.169 seconds
[2022-12-28 17:21:29,047] {processor.py:153} INFO - Started process (PID=10708) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:21:29,052] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:21:29,054] {logging_mixin.py:115} INFO - [2022-12-28 17:21:29,053] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:21:29,068] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:21:29,106] {logging_mixin.py:115} INFO - [2022-12-28 17:21:29,106] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:21:29,147] {logging_mixin.py:115} INFO - [2022-12-28 17:21:29,147] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:21:29,179] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.138 seconds
[2022-12-28 17:22:00,038] {processor.py:153} INFO - Started process (PID=10762) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:22:00,040] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:22:00,041] {logging_mixin.py:115} INFO - [2022-12-28 17:22:00,041] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:22:00,056] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:22:00,083] {logging_mixin.py:115} INFO - [2022-12-28 17:22:00,083] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:22:00,111] {logging_mixin.py:115} INFO - [2022-12-28 17:22:00,111] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:22:00,135] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.105 seconds
[2022-12-28 17:22:37,431] {processor.py:153} INFO - Started process (PID=10847) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:22:37,432] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:22:37,434] {logging_mixin.py:115} INFO - [2022-12-28 17:22:37,434] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:22:37,447] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:22:37,475] {logging_mixin.py:115} INFO - [2022-12-28 17:22:37,475] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:22:37,517] {logging_mixin.py:115} INFO - [2022-12-28 17:22:37,517] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:22:37,551] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.125 seconds
[2022-12-28 17:23:15,613] {processor.py:153} INFO - Started process (PID=10913) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:23:15,618] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:23:15,619] {logging_mixin.py:115} INFO - [2022-12-28 17:23:15,619] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:23:15,631] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:23:15,659] {logging_mixin.py:115} INFO - [2022-12-28 17:23:15,658] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:23:15,689] {logging_mixin.py:115} INFO - [2022-12-28 17:23:15,689] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:23:15,713] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.105 seconds
[2022-12-28 17:23:52,967] {processor.py:153} INFO - Started process (PID=10983) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:23:52,969] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:23:52,970] {logging_mixin.py:115} INFO - [2022-12-28 17:23:52,970] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:23:52,981] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:23:53,020] {logging_mixin.py:115} INFO - [2022-12-28 17:23:53,020] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:23:53,054] {logging_mixin.py:115} INFO - [2022-12-28 17:23:53,054] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:23:53,096] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.134 seconds
[2022-12-28 17:24:31,660] {processor.py:153} INFO - Started process (PID=11048) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:24:31,661] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:24:31,662] {logging_mixin.py:115} INFO - [2022-12-28 17:24:31,662] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:24:31,674] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:24:31,701] {logging_mixin.py:115} INFO - [2022-12-28 17:24:31,701] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:24:31,733] {logging_mixin.py:115} INFO - [2022-12-28 17:24:31,733] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:24:31,757] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.104 seconds
[2022-12-28 17:25:09,479] {processor.py:153} INFO - Started process (PID=11119) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:25:09,481] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:25:09,482] {logging_mixin.py:115} INFO - [2022-12-28 17:25:09,482] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:25:09,504] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:25:09,542] {logging_mixin.py:115} INFO - [2022-12-28 17:25:09,542] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:25:09,576] {logging_mixin.py:115} INFO - [2022-12-28 17:25:09,576] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:25:09,601] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.126 seconds
[2022-12-28 17:25:46,289] {processor.py:153} INFO - Started process (PID=11192) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:25:46,294] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:25:46,295] {logging_mixin.py:115} INFO - [2022-12-28 17:25:46,295] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:25:46,306] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:25:46,332] {logging_mixin.py:115} INFO - [2022-12-28 17:25:46,332] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:25:46,360] {logging_mixin.py:115} INFO - [2022-12-28 17:25:46,360] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:25:46,387] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.103 seconds
[2022-12-28 17:26:18,328] {processor.py:153} INFO - Started process (PID=11249) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:26:18,330] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:26:18,331] {logging_mixin.py:115} INFO - [2022-12-28 17:26:18,331] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:26:18,342] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:26:18,368] {logging_mixin.py:115} INFO - [2022-12-28 17:26:18,368] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:26:18,397] {logging_mixin.py:115} INFO - [2022-12-28 17:26:18,396] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:26:18,423] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.100 seconds
[2022-12-28 17:26:54,366] {processor.py:153} INFO - Started process (PID=11323) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:26:54,368] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:26:54,369] {logging_mixin.py:115} INFO - [2022-12-28 17:26:54,369] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:26:54,383] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:26:54,410] {logging_mixin.py:115} INFO - [2022-12-28 17:26:54,410] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:26:54,444] {logging_mixin.py:115} INFO - [2022-12-28 17:26:54,443] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:26:54,471] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.111 seconds
[2022-12-28 17:27:27,151] {processor.py:153} INFO - Started process (PID=11384) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:27:27,153] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:27:27,154] {logging_mixin.py:115} INFO - [2022-12-28 17:27:27,154] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:27:27,171] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:27:27,206] {logging_mixin.py:115} INFO - [2022-12-28 17:27:27,206] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:27:27,253] {logging_mixin.py:115} INFO - [2022-12-28 17:27:27,252] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:27:27,285] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.140 seconds
[2022-12-28 17:28:05,700] {processor.py:153} INFO - Started process (PID=11458) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:28:05,705] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:28:05,706] {logging_mixin.py:115} INFO - [2022-12-28 17:28:05,706] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:28:05,717] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:28:05,742] {logging_mixin.py:115} INFO - [2022-12-28 17:28:05,742] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:28:05,772] {logging_mixin.py:115} INFO - [2022-12-28 17:28:05,772] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:28:05,806] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.111 seconds
[2022-12-28 17:28:36,141] {processor.py:153} INFO - Started process (PID=11505) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:28:36,142] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:28:36,144] {logging_mixin.py:115} INFO - [2022-12-28 17:28:36,144] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:28:36,161] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:28:36,192] {logging_mixin.py:115} INFO - [2022-12-28 17:28:36,192] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:28:36,239] {logging_mixin.py:115} INFO - [2022-12-28 17:28:36,238] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:28:36,273] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.137 seconds
[2022-12-28 17:29:09,901] {processor.py:153} INFO - Started process (PID=11570) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:29:09,903] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:29:09,905] {logging_mixin.py:115} INFO - [2022-12-28 17:29:09,905] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:29:09,925] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:29:09,962] {logging_mixin.py:115} INFO - [2022-12-28 17:29:09,962] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:29:10,008] {logging_mixin.py:115} INFO - [2022-12-28 17:29:10,008] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:29:10,042] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.148 seconds
[2022-12-28 17:29:46,076] {processor.py:153} INFO - Started process (PID=11627) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:29:46,077] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:29:46,078] {logging_mixin.py:115} INFO - [2022-12-28 17:29:46,078] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:29:46,090] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:29:46,117] {logging_mixin.py:115} INFO - [2022-12-28 17:29:46,117] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:29:46,150] {logging_mixin.py:115} INFO - [2022-12-28 17:29:46,150] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:29:46,181] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.110 seconds
[2022-12-28 17:30:17,620] {processor.py:153} INFO - Started process (PID=11683) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:30:17,621] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:30:17,622] {logging_mixin.py:115} INFO - [2022-12-28 17:30:17,622] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:30:17,633] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:30:17,658] {logging_mixin.py:115} INFO - [2022-12-28 17:30:17,658] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:30:17,687] {logging_mixin.py:115} INFO - [2022-12-28 17:30:17,687] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:30:17,711] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.095 seconds
[2022-12-28 17:30:49,053] {processor.py:153} INFO - Started process (PID=11743) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:30:49,059] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:30:49,060] {logging_mixin.py:115} INFO - [2022-12-28 17:30:49,060] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:30:49,071] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:30:49,101] {logging_mixin.py:115} INFO - [2022-12-28 17:30:49,100] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:30:49,148] {logging_mixin.py:115} INFO - [2022-12-28 17:30:49,148] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:30:49,217] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.168 seconds
[2022-12-28 17:31:19,409] {processor.py:153} INFO - Started process (PID=11889) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:31:19,410] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:31:19,411] {logging_mixin.py:115} INFO - [2022-12-28 17:31:19,411] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:31:19,431] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:31:19,484] {logging_mixin.py:115} INFO - [2022-12-28 17:31:19,484] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:31:19,534] {logging_mixin.py:115} INFO - [2022-12-28 17:31:19,534] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:31:19,580] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.180 seconds
[2022-12-28 17:32:00,608] {processor.py:153} INFO - Started process (PID=11980) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:32:00,610] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:32:00,611] {logging_mixin.py:115} INFO - [2022-12-28 17:32:00,611] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:32:00,629] {logging_mixin.py:115} INFO - [2022-12-28 17:32:00,628] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 18, in <module>
    tarefa_2
NameError: name 'tarefa_2' is not defined
[2022-12-28 17:32:00,630] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:32:00,848] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.246 seconds
[2022-12-28 17:32:46,842] {processor.py:153} INFO - Started process (PID=12083) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:32:46,843] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:32:46,845] {logging_mixin.py:115} INFO - [2022-12-28 17:32:46,844] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:32:46,863] {logging_mixin.py:115} INFO - [2022-12-28 17:32:46,862] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 20, in <module>
    python_callable= extrai_dados()
NameError: name 'extrai_dados' is not defined
[2022-12-28 17:32:46,863] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:32:46,919] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.082 seconds
[2022-12-28 17:33:26,645] {processor.py:153} INFO - Started process (PID=12173) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:33:26,650] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:33:26,652] {logging_mixin.py:115} INFO - [2022-12-28 17:33:26,652] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:33:26,656] {logging_mixin.py:115} INFO - [2022-12-28 17:33:26,655] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 21
    tarefa_2 = PythonOperator(
    ^
IndentationError: expected an indented block
[2022-12-28 17:33:26,657] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:33:26,692] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.052 seconds
[2022-12-28 17:34:06,667] {processor.py:153} INFO - Started process (PID=12251) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:34:06,669] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:34:06,670] {logging_mixin.py:115} INFO - [2022-12-28 17:34:06,670] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:34:06,674] {logging_mixin.py:115} INFO - [2022-12-28 17:34:06,673] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 25
    tarefa_2 = PythonOperator(
    ^
IndentationError: expected an indented block
[2022-12-28 17:34:06,675] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:34:06,713] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.051 seconds
[2022-12-28 17:34:49,649] {processor.py:153} INFO - Started process (PID=12333) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:34:49,653] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:34:49,654] {logging_mixin.py:115} INFO - [2022-12-28 17:34:49,654] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:34:49,669] {logging_mixin.py:115} INFO - [2022-12-28 17:34:49,668] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2022-12-28 17:34:49,670] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:34:49,705] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.062 seconds
[2022-12-28 17:35:28,952] {processor.py:153} INFO - Started process (PID=12407) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:35:28,976] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:35:28,977] {logging_mixin.py:115} INFO - [2022-12-28 17:35:28,977] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:35:28,992] {logging_mixin.py:115} INFO - [2022-12-28 17:35:28,991] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2022-12-28 17:35:29,001] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:35:29,055] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.108 seconds
[2022-12-28 17:36:08,816] {processor.py:153} INFO - Started process (PID=12485) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:36:08,817] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:36:08,818] {logging_mixin.py:115} INFO - [2022-12-28 17:36:08,818] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:36:08,832] {logging_mixin.py:115} INFO - [2022-12-28 17:36:08,831] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
[2022-12-28 17:36:08,833] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:36:08,867] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.057 seconds
[2022-12-28 17:36:48,169] {processor.py:153} INFO - Started process (PID=12570) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:36:48,174] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:36:48,175] {logging_mixin.py:115} INFO - [2022-12-28 17:36:48,175] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:36:48,180] {logging_mixin.py:115} INFO - [2022-12-28 17:36:48,179] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op
    ^
SyntaxError: invalid syntax
[2022-12-28 17:36:48,181] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:36:48,213] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.049 seconds
[2022-12-28 17:37:20,464] {processor.py:153} INFO - Started process (PID=12637) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:37:20,466] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:37:20,467] {logging_mixin.py:115} INFO - [2022-12-28 17:37:20,467] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:37:20,472] {logging_mixin.py:115} INFO - [2022-12-28 17:37:20,471] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:37:20,473] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:37:20,568] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.121 seconds
[2022-12-28 17:37:56,121] {processor.py:153} INFO - Started process (PID=12733) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:37:56,123] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:37:56,125] {logging_mixin.py:115} INFO - [2022-12-28 17:37:56,124] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:37:56,131] {logging_mixin.py:115} INFO - [2022-12-28 17:37:56,129] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:37:56,131] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:37:56,181] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.066 seconds
[2022-12-28 17:38:26,883] {processor.py:153} INFO - Started process (PID=12798) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:38:26,883] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:38:26,884] {logging_mixin.py:115} INFO - [2022-12-28 17:38:26,884] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:38:26,891] {logging_mixin.py:115} INFO - [2022-12-28 17:38:26,890] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:38:26,892] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:38:26,951] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.073 seconds
[2022-12-28 17:39:06,550] {processor.py:153} INFO - Started process (PID=12875) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:39:06,554] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:39:06,555] {logging_mixin.py:115} INFO - [2022-12-28 17:39:06,555] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:39:06,561] {logging_mixin.py:115} INFO - [2022-12-28 17:39:06,560] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:39:06,562] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:39:06,597] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.052 seconds
[2022-12-28 17:39:42,265] {processor.py:153} INFO - Started process (PID=12947) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:39:42,266] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:39:42,267] {logging_mixin.py:115} INFO - [2022-12-28 17:39:42,267] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:39:42,272] {logging_mixin.py:115} INFO - [2022-12-28 17:39:42,271] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:39:42,273] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:39:42,307] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.047 seconds
[2022-12-28 17:40:17,574] {processor.py:153} INFO - Started process (PID=13014) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:40:17,577] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:40:17,578] {logging_mixin.py:115} INFO - [2022-12-28 17:40:17,578] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:40:17,584] {logging_mixin.py:115} INFO - [2022-12-28 17:40:17,582] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:40:17,584] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:40:17,679] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.123 seconds
[2022-12-28 17:40:58,447] {processor.py:153} INFO - Started process (PID=13095) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:40:58,459] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:40:58,460] {logging_mixin.py:115} INFO - [2022-12-28 17:40:58,460] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:40:58,464] {logging_mixin.py:115} INFO - [2022-12-28 17:40:58,463] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 5
    from airflow.macros import
                              ^
SyntaxError: invalid syntax
[2022-12-28 17:40:58,464] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:40:58,592] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.149 seconds
[2022-12-28 17:41:34,594] {processor.py:153} INFO - Started process (PID=13160) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:41:34,598] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:41:34,599] {logging_mixin.py:115} INFO - [2022-12-28 17:41:34,599] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:41:34,604] {logging_mixin.py:115} INFO - [2022-12-28 17:41:34,603] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:41:34,605] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:41:34,637] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.050 seconds
[2022-12-28 17:42:08,654] {processor.py:153} INFO - Started process (PID=13222) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:42:08,655] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:42:08,656] {logging_mixin.py:115} INFO - [2022-12-28 17:42:08,656] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:42:08,661] {logging_mixin.py:115} INFO - [2022-12-28 17:42:08,660] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:42:08,662] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:42:08,697] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.047 seconds
[2022-12-28 17:42:39,304] {processor.py:153} INFO - Started process (PID=13285) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:42:39,305] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:42:39,306] {logging_mixin.py:115} INFO - [2022-12-28 17:42:39,306] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:42:39,311] {logging_mixin.py:115} INFO - [2022-12-28 17:42:39,310] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:42:39,311] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:42:39,348] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.049 seconds
[2022-12-28 17:43:10,851] {processor.py:153} INFO - Started process (PID=13338) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:43:10,855] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:43:10,857] {logging_mixin.py:115} INFO - [2022-12-28 17:43:10,857] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:43:10,865] {logging_mixin.py:115} INFO - [2022-12-28 17:43:10,864] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:43:10,866] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:43:10,899] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.053 seconds
[2022-12-28 17:43:49,127] {processor.py:153} INFO - Started process (PID=13411) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:43:49,131] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:43:49,132] {logging_mixin.py:115} INFO - [2022-12-28 17:43:49,132] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:43:49,138] {logging_mixin.py:115} INFO - [2022-12-28 17:43:49,136] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:43:49,138] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:43:49,174] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.052 seconds
[2022-12-28 17:44:23,952] {processor.py:153} INFO - Started process (PID=13480) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:44:23,954] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:44:23,955] {logging_mixin.py:115} INFO - [2022-12-28 17:44:23,954] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:44:23,959] {logging_mixin.py:115} INFO - [2022-12-28 17:44:23,958] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:44:23,960] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:44:23,993] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.046 seconds
[2022-12-28 17:44:57,251] {processor.py:153} INFO - Started process (PID=13545) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:44:57,257] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:44:57,259] {logging_mixin.py:115} INFO - [2022-12-28 17:44:57,258] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:44:57,280] {logging_mixin.py:115} INFO - [2022-12-28 17:44:57,278] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 42
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:44:57,280] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:44:57,314] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.070 seconds
[2022-12-28 17:45:32,894] {processor.py:153} INFO - Started process (PID=13629) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:45:32,902] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:45:32,905] {logging_mixin.py:115} INFO - [2022-12-28 17:45:32,904] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:45:32,913] {logging_mixin.py:115} INFO - [2022-12-28 17:45:32,910] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:45:32,914] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:45:33,008] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.129 seconds
[2022-12-28 17:46:11,883] {processor.py:153} INFO - Started process (PID=13825) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:46:11,906] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:46:11,907] {logging_mixin.py:115} INFO - [2022-12-28 17:46:11,907] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:46:11,913] {logging_mixin.py:115} INFO - [2022-12-28 17:46:11,912] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:46:11,924] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:46:12,049] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.170 seconds
[2022-12-28 17:46:56,035] {processor.py:153} INFO - Started process (PID=13924) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:46:56,040] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:46:56,041] {logging_mixin.py:115} INFO - [2022-12-28 17:46:56,041] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:46:56,045] {logging_mixin.py:115} INFO - [2022-12-28 17:46:56,044] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 41
    op_kwargs= {'data_interval_end':'{{data_interval_end.strftime("%Y-%m_%d)}}'}
    ^
SyntaxError: invalid syntax
[2022-12-28 17:46:56,046] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:46:56,079] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.049 seconds
[2022-12-28 17:47:30,350] {processor.py:153} INFO - Started process (PID=13986) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:47:30,352] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:47:30,353] {logging_mixin.py:115} INFO - [2022-12-28 17:47:30,352] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:47:31,067] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:47:31,134] {logging_mixin.py:115} INFO - [2022-12-28 17:47:31,134] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:47:31,157] {logging_mixin.py:115} INFO - [2022-12-28 17:47:31,156] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:47:31,218] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.873 seconds
[2022-12-28 17:48:12,090] {processor.py:153} INFO - Started process (PID=14067) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:48:12,095] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:48:12,096] {logging_mixin.py:115} INFO - [2022-12-28 17:48:12,095] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:48:12,882] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:48:12,917] {logging_mixin.py:115} INFO - [2022-12-28 17:48:12,916] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:48:12,942] {logging_mixin.py:115} INFO - [2022-12-28 17:48:12,942] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 17:48:12,969] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.884 seconds
[2022-12-28 17:49:02,311] {processor.py:153} INFO - Started process (PID=14165) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:49:02,316] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:49:02,317] {logging_mixin.py:115} INFO - [2022-12-28 17:49:02,317] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:49:03,180] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:49:03,211] {logging_mixin.py:115} INFO - [2022-12-28 17:49:03,210] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:49:03,273] {logging_mixin.py:115} INFO - [2022-12-28 17:49:03,273] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-09-05T00:00:00+00:00, run_after=2022-09-12T00:00:00+00:00
[2022-12-28 17:49:03,349] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.060 seconds
[2022-12-28 17:50:22,542] {processor.py:153} INFO - Started process (PID=14309) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:50:22,549] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:50:22,551] {logging_mixin.py:115} INFO - [2022-12-28 17:50:22,551] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:50:23,562] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:50:23,620] {logging_mixin.py:115} INFO - [2022-12-28 17:50:23,620] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:50:23,690] {logging_mixin.py:115} INFO - [2022-12-28 17:50:23,690] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-05T00:00:00+00:00, run_after=2022-12-12T00:00:00+00:00
[2022-12-28 17:50:23,742] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.207 seconds
[2022-12-28 17:51:15,085] {processor.py:153} INFO - Started process (PID=14393) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:51:15,089] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:51:15,090] {logging_mixin.py:115} INFO - [2022-12-28 17:51:15,090] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:51:15,902] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:51:15,925] {logging_mixin.py:115} INFO - [2022-12-28 17:51:15,925] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:51:15,955] {logging_mixin.py:115} INFO - [2022-12-28 17:51:15,955] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:51:15,989] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.911 seconds
[2022-12-28 17:51:52,806] {processor.py:153} INFO - Started process (PID=14457) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:51:52,809] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:51:52,810] {logging_mixin.py:115} INFO - [2022-12-28 17:51:52,810] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:51:54,024] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:51:54,067] {logging_mixin.py:115} INFO - [2022-12-28 17:51:54,066] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:51:54,132] {logging_mixin.py:115} INFO - [2022-12-28 17:51:54,131] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:51:54,160] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.363 seconds
[2022-12-28 17:52:32,798] {processor.py:153} INFO - Started process (PID=14530) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:52:32,799] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:52:32,800] {logging_mixin.py:115} INFO - [2022-12-28 17:52:32,800] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:52:33,522] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:52:33,542] {logging_mixin.py:115} INFO - [2022-12-28 17:52:33,542] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:52:33,572] {logging_mixin.py:115} INFO - [2022-12-28 17:52:33,572] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:52:33,607] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.814 seconds
[2022-12-28 17:53:10,322] {processor.py:153} INFO - Started process (PID=14594) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:53:10,327] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:53:10,329] {logging_mixin.py:115} INFO - [2022-12-28 17:53:10,329] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:53:11,052] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:53:11,074] {logging_mixin.py:115} INFO - [2022-12-28 17:53:11,074] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:53:11,102] {logging_mixin.py:115} INFO - [2022-12-28 17:53:11,102] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:53:11,139] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.821 seconds
[2022-12-28 17:53:47,718] {processor.py:153} INFO - Started process (PID=14659) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:53:47,721] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:53:47,722] {logging_mixin.py:115} INFO - [2022-12-28 17:53:47,722] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:53:48,334] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:53:48,356] {logging_mixin.py:115} INFO - [2022-12-28 17:53:48,356] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:53:48,380] {logging_mixin.py:115} INFO - [2022-12-28 17:53:48,380] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:53:48,430] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.718 seconds
[2022-12-28 17:54:26,508] {processor.py:153} INFO - Started process (PID=14728) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:54:26,512] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:54:26,516] {logging_mixin.py:115} INFO - [2022-12-28 17:54:26,516] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:54:27,148] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:54:27,168] {logging_mixin.py:115} INFO - [2022-12-28 17:54:27,168] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:54:27,194] {logging_mixin.py:115} INFO - [2022-12-28 17:54:27,194] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:54:27,229] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.727 seconds
[2022-12-28 17:55:06,178] {processor.py:153} INFO - Started process (PID=14800) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:55:06,185] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:55:06,187] {logging_mixin.py:115} INFO - [2022-12-28 17:55:06,186] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:55:07,136] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:55:07,166] {logging_mixin.py:115} INFO - [2022-12-28 17:55:07,165] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:55:07,193] {logging_mixin.py:115} INFO - [2022-12-28 17:55:07,193] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:55:07,230] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.057 seconds
[2022-12-28 17:55:50,475] {processor.py:153} INFO - Started process (PID=14886) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:55:50,483] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:55:50,484] {logging_mixin.py:115} INFO - [2022-12-28 17:55:50,484] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:55:50,493] {logging_mixin.py:115} INFO - [2022-12-28 17:55:50,487] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 29
    https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/[location]/[date1]/[date2]?key=YOUR_API_KEY 
IndentationError: unexpected indent
[2022-12-28 17:55:50,494] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:55:50,561] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.095 seconds
[2022-12-28 17:56:32,472] {processor.py:153} INFO - Started process (PID=14954) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:56:32,478] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:56:32,480] {logging_mixin.py:115} INFO - [2022-12-28 17:56:32,480] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:56:32,489] {logging_mixin.py:115} INFO - [2022-12-28 17:56:32,487] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 846, in exec_module
  File "<frozen importlib._bootstrap_external>", line 983, in get_code
  File "<frozen importlib._bootstrap_external>", line 913, in source_to_code
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 29
    https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/[location]/[date1]/[date2]?key=YOUR_API_KEY 
IndentationError: unexpected indent
[2022-12-28 17:56:32,489] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:56:32,544] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.079 seconds
[2022-12-28 17:57:19,099] {processor.py:153} INFO - Started process (PID=15042) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:57:19,104] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:57:19,105] {logging_mixin.py:115} INFO - [2022-12-28 17:57:19,105] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:57:20,005] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:57:20,023] {logging_mixin.py:115} INFO - [2022-12-28 17:57:20,023] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:57:20,059] {logging_mixin.py:115} INFO - [2022-12-28 17:57:20,059] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:57:20,146] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.056 seconds
[2022-12-28 17:57:51,489] {processor.py:153} INFO - Started process (PID=15094) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:57:51,492] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:57:51,496] {logging_mixin.py:115} INFO - [2022-12-28 17:57:51,496] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:57:52,806] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:57:52,846] {logging_mixin.py:115} INFO - [2022-12-28 17:57:52,846] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:57:52,916] {logging_mixin.py:115} INFO - [2022-12-28 17:57:52,916] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:57:52,991] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.539 seconds
[2022-12-28 17:58:24,190] {processor.py:153} INFO - Started process (PID=15150) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:58:24,192] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:58:24,193] {logging_mixin.py:115} INFO - [2022-12-28 17:58:24,193] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:58:24,853] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:58:24,870] {logging_mixin.py:115} INFO - [2022-12-28 17:58:24,869] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:58:24,892] {logging_mixin.py:115} INFO - [2022-12-28 17:58:24,892] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:58:24,917] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.731 seconds
[2022-12-28 17:59:00,146] {processor.py:153} INFO - Started process (PID=15228) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:59:00,151] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:59:00,152] {logging_mixin.py:115} INFO - [2022-12-28 17:59:00,152] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:59:00,820] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:59:00,836] {logging_mixin.py:115} INFO - [2022-12-28 17:59:00,836] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:59:00,861] {logging_mixin.py:115} INFO - [2022-12-28 17:59:00,861] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:59:01,016] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.874 seconds
[2022-12-28 17:59:36,421] {processor.py:153} INFO - Started process (PID=15286) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:59:36,422] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 17:59:36,423] {logging_mixin.py:115} INFO - [2022-12-28 17:59:36,423] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:59:37,083] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 17:59:37,101] {logging_mixin.py:115} INFO - [2022-12-28 17:59:37,101] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 17:59:37,125] {logging_mixin.py:115} INFO - [2022-12-28 17:59:37,125] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 17:59:37,149] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.733 seconds
[2022-12-28 18:00:08,784] {processor.py:153} INFO - Started process (PID=15346) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:00:08,786] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:00:08,787] {logging_mixin.py:115} INFO - [2022-12-28 18:00:08,787] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:00:09,468] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:00:09,485] {logging_mixin.py:115} INFO - [2022-12-28 18:00:09,485] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:00:09,510] {logging_mixin.py:115} INFO - [2022-12-28 18:00:09,510] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:00:09,533] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.754 seconds
[2022-12-28 18:00:40,586] {processor.py:153} INFO - Started process (PID=15402) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:00:40,587] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:00:40,588] {logging_mixin.py:115} INFO - [2022-12-28 18:00:40,588] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:00:41,259] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:00:41,277] {logging_mixin.py:115} INFO - [2022-12-28 18:00:41,276] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:00:41,303] {logging_mixin.py:115} INFO - [2022-12-28 18:00:41,302] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:00:41,327] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.746 seconds
[2022-12-28 18:01:18,977] {processor.py:153} INFO - Started process (PID=15472) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:01:18,982] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:01:18,983] {logging_mixin.py:115} INFO - [2022-12-28 18:01:18,983] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:01:19,817] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:01:19,835] {logging_mixin.py:115} INFO - [2022-12-28 18:01:19,835] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:01:19,876] {logging_mixin.py:115} INFO - [2022-12-28 18:01:19,876] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:01:19,955] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.996 seconds
[2022-12-28 18:02:04,486] {processor.py:153} INFO - Started process (PID=15548) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:02:04,490] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:02:04,491] {logging_mixin.py:115} INFO - [2022-12-28 18:02:04,491] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:02:05,206] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:02:05,225] {logging_mixin.py:115} INFO - [2022-12-28 18:02:05,225] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:02:05,250] {logging_mixin.py:115} INFO - [2022-12-28 18:02:05,250] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:02:05,275] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.794 seconds
[2022-12-28 18:02:38,605] {processor.py:153} INFO - Started process (PID=15616) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:02:38,607] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:02:38,607] {logging_mixin.py:115} INFO - [2022-12-28 18:02:38,607] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:02:39,295] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:02:39,314] {logging_mixin.py:115} INFO - [2022-12-28 18:02:39,313] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:02:39,336] {logging_mixin.py:115} INFO - [2022-12-28 18:02:39,336] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:02:39,359] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.759 seconds
[2022-12-28 18:03:19,479] {processor.py:153} INFO - Started process (PID=15685) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:03:19,484] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:03:19,485] {logging_mixin.py:115} INFO - [2022-12-28 18:03:19,485] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:03:20,138] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:03:20,153] {logging_mixin.py:115} INFO - [2022-12-28 18:03:20,153] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:03:20,185] {logging_mixin.py:115} INFO - [2022-12-28 18:03:20,184] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:03:20,209] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.734 seconds
[2022-12-28 18:03:51,420] {processor.py:153} INFO - Started process (PID=15752) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:03:51,421] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:03:51,422] {logging_mixin.py:115} INFO - [2022-12-28 18:03:51,422] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:03:52,061] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:03:52,080] {logging_mixin.py:115} INFO - [2022-12-28 18:03:52,079] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:03:52,103] {logging_mixin.py:115} INFO - [2022-12-28 18:03:52,103] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:03:52,126] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.711 seconds
[2022-12-28 18:04:30,360] {processor.py:153} INFO - Started process (PID=15838) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:04:30,365] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:04:30,366] {logging_mixin.py:115} INFO - [2022-12-28 18:04:30,366] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:04:31,258] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:04:31,292] {logging_mixin.py:115} INFO - [2022-12-28 18:04:31,292] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:04:31,332] {logging_mixin.py:115} INFO - [2022-12-28 18:04:31,331] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:04:31,477] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.128 seconds
[2022-12-28 18:05:19,402] {processor.py:153} INFO - Started process (PID=15918) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:05:19,413] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:05:19,414] {logging_mixin.py:115} INFO - [2022-12-28 18:05:19,414] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:05:20,552] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:05:20,639] {logging_mixin.py:115} INFO - [2022-12-28 18:05:20,639] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:05:20,659] {logging_mixin.py:115} INFO - [2022-12-28 18:05:20,658] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 18:05:20,679] {logging_mixin.py:115} INFO - [2022-12-28 18:05:20,679] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:05:20,725] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.334 seconds
[2022-12-28 18:05:54,425] {processor.py:153} INFO - Started process (PID=15970) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:05:54,435] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:05:54,436] {logging_mixin.py:115} INFO - [2022-12-28 18:05:54,436] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:05:55,443] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:05:55,481] {logging_mixin.py:115} INFO - [2022-12-28 18:05:55,478] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:05:55,518] {logging_mixin.py:115} INFO - [2022-12-28 18:05:55,517] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:05:55,548] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.128 seconds
[2022-12-28 18:06:29,927] {processor.py:153} INFO - Started process (PID=16028) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:06:29,931] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:06:29,932] {logging_mixin.py:115} INFO - [2022-12-28 18:06:29,932] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:06:30,688] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:06:30,710] {logging_mixin.py:115} INFO - [2022-12-28 18:06:30,710] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:06:30,741] {logging_mixin.py:115} INFO - [2022-12-28 18:06:30,740] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:06:30,822] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.899 seconds
[2022-12-28 18:07:12,851] {processor.py:153} INFO - Started process (PID=16102) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:07:12,853] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:07:12,853] {logging_mixin.py:115} INFO - [2022-12-28 18:07:12,853] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:07:13,659] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:07:13,680] {logging_mixin.py:115} INFO - [2022-12-28 18:07:13,679] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:07:13,707] {logging_mixin.py:115} INFO - [2022-12-28 18:07:13,706] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:07:13,738] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.894 seconds
[2022-12-28 18:07:51,253] {processor.py:153} INFO - Started process (PID=16169) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:07:51,255] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:07:51,256] {logging_mixin.py:115} INFO - [2022-12-28 18:07:51,256] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:07:52,056] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:07:52,089] {logging_mixin.py:115} INFO - [2022-12-28 18:07:52,088] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:07:52,121] {logging_mixin.py:115} INFO - [2022-12-28 18:07:52,121] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:07:52,150] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.902 seconds
[2022-12-28 18:08:26,658] {processor.py:153} INFO - Started process (PID=16241) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:08:27,358] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:08:27,361] {logging_mixin.py:115} INFO - [2022-12-28 18:08:27,360] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:08:28,123] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:08:28,143] {logging_mixin.py:115} INFO - [2022-12-28 18:08:28,142] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:08:28,174] {logging_mixin.py:115} INFO - [2022-12-28 18:08:28,173] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:08:28,205] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.562 seconds
[2022-12-28 18:09:04,909] {processor.py:153} INFO - Started process (PID=16310) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:09:04,910] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:09:04,911] {logging_mixin.py:115} INFO - [2022-12-28 18:09:04,911] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:09:05,693] {logging_mixin.py:115} INFO - [2022-12-28 18:09:05,690] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 38, in <module>
    tarefa_2 = PythonOperator(
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 390, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 158, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-12-28 18:09:05,694] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:09:05,741] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.841 seconds
[2022-12-28 18:09:41,955] {processor.py:153} INFO - Started process (PID=16369) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:09:41,956] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:09:41,960] {logging_mixin.py:115} INFO - [2022-12-28 18:09:41,960] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:09:42,741] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:09:42,758] {logging_mixin.py:115} INFO - [2022-12-28 18:09:42,758] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:09:42,785] {logging_mixin.py:115} INFO - [2022-12-28 18:09:42,785] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:09:42,888] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.938 seconds
[2022-12-28 18:10:21,988] {processor.py:153} INFO - Started process (PID=16439) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:10:22,001] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:10:22,002] {logging_mixin.py:115} INFO - [2022-12-28 18:10:22,002] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:10:22,716] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:10:22,735] {logging_mixin.py:115} INFO - [2022-12-28 18:10:22,734] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:10:22,760] {logging_mixin.py:115} INFO - [2022-12-28 18:10:22,759] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:10:22,783] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.803 seconds
[2022-12-28 18:11:01,082] {processor.py:153} INFO - Started process (PID=16509) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:11:01,084] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:11:01,085] {logging_mixin.py:115} INFO - [2022-12-28 18:11:01,085] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:11:01,828] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:11:01,847] {logging_mixin.py:115} INFO - [2022-12-28 18:11:01,847] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:11:01,874] {logging_mixin.py:115} INFO - [2022-12-28 18:11:01,873] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:11:01,907] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.830 seconds
[2022-12-28 18:11:42,425] {processor.py:153} INFO - Started process (PID=16572) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:11:42,426] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:11:42,427] {logging_mixin.py:115} INFO - [2022-12-28 18:11:42,427] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:11:43,222] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:11:43,238] {logging_mixin.py:115} INFO - [2022-12-28 18:11:43,237] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:11:43,261] {logging_mixin.py:115} INFO - [2022-12-28 18:11:43,261] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:11:43,329] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.915 seconds
[2022-12-28 18:12:16,882] {processor.py:153} INFO - Started process (PID=16633) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:12:16,892] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:12:16,894] {logging_mixin.py:115} INFO - [2022-12-28 18:12:16,894] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:12:17,604] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:12:17,620] {logging_mixin.py:115} INFO - [2022-12-28 18:12:17,620] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:12:17,648] {logging_mixin.py:115} INFO - [2022-12-28 18:12:17,648] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:12:17,711] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.834 seconds
[2022-12-28 18:12:55,099] {processor.py:153} INFO - Started process (PID=16705) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:12:55,100] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:12:55,101] {logging_mixin.py:115} INFO - [2022-12-28 18:12:55,101] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:12:55,885] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:12:55,934] {logging_mixin.py:115} INFO - [2022-12-28 18:12:55,934] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:12:55,980] {logging_mixin.py:115} INFO - [2022-12-28 18:12:55,980] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:12:56,033] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.941 seconds
[2022-12-28 18:14:03,296] {processor.py:153} INFO - Started process (PID=16823) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:14:03,299] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:14:03,302] {logging_mixin.py:115} INFO - [2022-12-28 18:14:03,302] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:14:03,979] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:14:03,996] {logging_mixin.py:115} INFO - [2022-12-28 18:14:03,995] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:14:04,032] {logging_mixin.py:115} INFO - [2022-12-28 18:14:04,031] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-09-19T00:00:00+00:00, run_after=2022-09-26T00:00:00+00:00
[2022-12-28 18:14:04,118] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.828 seconds
[2022-12-28 18:15:20,381] {processor.py:153} INFO - Started process (PID=16980) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:15:20,386] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:15:20,387] {logging_mixin.py:115} INFO - [2022-12-28 18:15:20,387] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:15:21,081] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:15:21,100] {logging_mixin.py:115} INFO - [2022-12-28 18:15:21,100] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:15:21,123] {logging_mixin.py:115} INFO - [2022-12-28 18:15:21,123] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:15:21,156] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.780 seconds
[2022-12-28 18:15:52,786] {processor.py:153} INFO - Started process (PID=17046) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:15:52,787] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:15:52,788] {logging_mixin.py:115} INFO - [2022-12-28 18:15:52,788] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:15:53,420] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:15:53,436] {logging_mixin.py:115} INFO - [2022-12-28 18:15:53,436] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:15:53,459] {logging_mixin.py:115} INFO - [2022-12-28 18:15:53,459] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:15:53,482] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.701 seconds
[2022-12-28 18:16:23,868] {processor.py:153} INFO - Started process (PID=17107) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:16:23,869] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:16:23,870] {logging_mixin.py:115} INFO - [2022-12-28 18:16:23,869] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:16:24,478] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:16:24,495] {logging_mixin.py:115} INFO - [2022-12-28 18:16:24,494] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:16:24,522] {logging_mixin.py:115} INFO - [2022-12-28 18:16:24,522] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:16:24,554] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.691 seconds
[2022-12-28 18:17:03,827] {processor.py:153} INFO - Started process (PID=17175) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:17:03,831] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:17:03,832] {logging_mixin.py:115} INFO - [2022-12-28 18:17:03,832] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:17:04,541] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:17:04,562] {logging_mixin.py:115} INFO - [2022-12-28 18:17:04,562] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:17:04,588] {logging_mixin.py:115} INFO - [2022-12-28 18:17:04,588] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:17:04,615] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.793 seconds
[2022-12-28 18:17:41,673] {processor.py:153} INFO - Started process (PID=17248) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:17:41,678] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:17:41,679] {logging_mixin.py:115} INFO - [2022-12-28 18:17:41,678] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:17:42,348] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:17:42,365] {logging_mixin.py:115} INFO - [2022-12-28 18:17:42,365] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:17:42,391] {logging_mixin.py:115} INFO - [2022-12-28 18:17:42,391] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:17:42,416] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.748 seconds
[2022-12-28 18:18:20,696] {processor.py:153} INFO - Started process (PID=17314) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:18:20,701] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:18:20,702] {logging_mixin.py:115} INFO - [2022-12-28 18:18:20,702] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:18:21,344] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:18:21,363] {logging_mixin.py:115} INFO - [2022-12-28 18:18:21,363] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:18:21,387] {logging_mixin.py:115} INFO - [2022-12-28 18:18:21,386] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:18:21,421] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.730 seconds
[2022-12-28 18:18:52,274] {processor.py:153} INFO - Started process (PID=17372) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:18:52,276] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:18:52,277] {logging_mixin.py:115} INFO - [2022-12-28 18:18:52,277] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:18:52,896] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:18:52,913] {logging_mixin.py:115} INFO - [2022-12-28 18:18:52,913] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:18:52,938] {logging_mixin.py:115} INFO - [2022-12-28 18:18:52,938] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:18:52,965] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.697 seconds
[2022-12-28 18:19:23,677] {processor.py:153} INFO - Started process (PID=17433) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:19:23,678] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:19:23,679] {logging_mixin.py:115} INFO - [2022-12-28 18:19:23,679] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:19:24,307] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:19:24,324] {logging_mixin.py:115} INFO - [2022-12-28 18:19:24,323] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:19:24,347] {logging_mixin.py:115} INFO - [2022-12-28 18:19:24,347] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:19:24,372] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.700 seconds
[2022-12-28 18:19:55,056] {processor.py:153} INFO - Started process (PID=17496) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:19:55,062] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:19:55,063] {logging_mixin.py:115} INFO - [2022-12-28 18:19:55,063] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:19:55,759] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:19:55,777] {logging_mixin.py:115} INFO - [2022-12-28 18:19:55,777] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:19:55,802] {logging_mixin.py:115} INFO - [2022-12-28 18:19:55,802] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:19:55,828] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.777 seconds
[2022-12-28 18:20:34,424] {processor.py:153} INFO - Started process (PID=17574) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:20:34,439] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:20:34,441] {logging_mixin.py:115} INFO - [2022-12-28 18:20:34,440] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:20:35,069] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:20:35,092] {logging_mixin.py:115} INFO - [2022-12-28 18:20:35,091] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:20:35,116] {logging_mixin.py:115} INFO - [2022-12-28 18:20:35,116] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:20:35,161] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.741 seconds
[2022-12-28 18:21:05,937] {processor.py:153} INFO - Started process (PID=17635) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:21:05,938] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:21:05,939] {logging_mixin.py:115} INFO - [2022-12-28 18:21:05,939] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:21:06,779] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:21:06,795] {logging_mixin.py:115} INFO - [2022-12-28 18:21:06,794] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:21:06,818] {logging_mixin.py:115} INFO - [2022-12-28 18:21:06,818] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:21:06,876] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.951 seconds
[2022-12-28 18:21:39,185] {processor.py:153} INFO - Started process (PID=17710) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:21:39,187] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:21:39,188] {logging_mixin.py:115} INFO - [2022-12-28 18:21:39,188] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:21:40,054] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:21:40,070] {logging_mixin.py:115} INFO - [2022-12-28 18:21:40,070] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:21:40,096] {logging_mixin.py:115} INFO - [2022-12-28 18:21:40,095] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:21:40,160] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.983 seconds
[2022-12-28 18:22:13,865] {processor.py:153} INFO - Started process (PID=17772) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:22:13,870] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:22:13,871] {logging_mixin.py:115} INFO - [2022-12-28 18:22:13,871] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:22:14,534] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:22:14,551] {logging_mixin.py:115} INFO - [2022-12-28 18:22:14,551] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:22:14,576] {logging_mixin.py:115} INFO - [2022-12-28 18:22:14,576] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:22:14,600] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.740 seconds
[2022-12-28 18:22:52,110] {processor.py:153} INFO - Started process (PID=17854) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:22:52,111] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:22:52,112] {logging_mixin.py:115} INFO - [2022-12-28 18:22:52,112] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:22:52,793] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:22:52,811] {logging_mixin.py:115} INFO - [2022-12-28 18:22:52,811] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:22:52,837] {logging_mixin.py:115} INFO - [2022-12-28 18:22:52,837] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:22:52,865] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.760 seconds
[2022-12-28 18:23:23,415] {processor.py:153} INFO - Started process (PID=17906) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:23:23,416] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:23:23,417] {logging_mixin.py:115} INFO - [2022-12-28 18:23:23,417] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:23:24,121] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:23:24,138] {logging_mixin.py:115} INFO - [2022-12-28 18:23:24,138] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:23:24,163] {logging_mixin.py:115} INFO - [2022-12-28 18:23:24,163] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:23:24,200] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.793 seconds
[2022-12-28 18:24:01,816] {processor.py:153} INFO - Started process (PID=17977) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:24:01,820] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:24:01,821] {logging_mixin.py:115} INFO - [2022-12-28 18:24:01,821] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:24:02,503] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:24:02,520] {logging_mixin.py:115} INFO - [2022-12-28 18:24:02,520] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:24:02,545] {logging_mixin.py:115} INFO - [2022-12-28 18:24:02,545] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:24:02,572] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.761 seconds
[2022-12-28 18:24:33,774] {processor.py:153} INFO - Started process (PID=18031) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:24:33,775] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:24:33,777] {logging_mixin.py:115} INFO - [2022-12-28 18:24:33,777] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:24:34,406] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:24:34,436] {logging_mixin.py:115} INFO - [2022-12-28 18:24:34,435] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:24:34,462] {logging_mixin.py:115} INFO - [2022-12-28 18:24:34,461] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:24:34,487] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.718 seconds
[2022-12-28 18:25:04,656] {processor.py:153} INFO - Started process (PID=18086) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:25:04,657] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:25:04,658] {logging_mixin.py:115} INFO - [2022-12-28 18:25:04,658] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:25:05,325] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:25:05,341] {logging_mixin.py:115} INFO - [2022-12-28 18:25:05,340] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:25:05,363] {logging_mixin.py:115} INFO - [2022-12-28 18:25:05,363] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:25:05,387] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.736 seconds
[2022-12-28 18:25:35,467] {processor.py:153} INFO - Started process (PID=18143) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:25:35,468] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:25:35,470] {logging_mixin.py:115} INFO - [2022-12-28 18:25:35,470] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:25:36,453] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:25:36,532] {logging_mixin.py:115} INFO - [2022-12-28 18:25:36,532] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:25:36,569] {logging_mixin.py:115} INFO - [2022-12-28 18:25:36,569] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:25:36,602] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.145 seconds
[2022-12-28 18:26:08,447] {processor.py:153} INFO - Started process (PID=18195) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:26:08,454] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:26:08,456] {logging_mixin.py:115} INFO - [2022-12-28 18:26:08,455] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:26:09,663] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:26:09,726] {logging_mixin.py:115} INFO - [2022-12-28 18:26:09,725] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:26:09,820] {logging_mixin.py:115} INFO - [2022-12-28 18:26:09,820] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:26:09,919] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.485 seconds
[2022-12-28 18:26:52,628] {processor.py:153} INFO - Started process (PID=18262) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:26:52,636] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:26:52,637] {logging_mixin.py:115} INFO - [2022-12-28 18:26:52,637] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:26:53,481] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:26:53,576] {logging_mixin.py:115} INFO - [2022-12-28 18:26:53,576] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:26:53,587] {logging_mixin.py:115} INFO - [2022-12-28 18:26:53,586] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 18:26:53,604] {logging_mixin.py:115} INFO - [2022-12-28 18:26:53,604] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:26:53,637] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.020 seconds
[2022-12-28 18:27:28,752] {processor.py:153} INFO - Started process (PID=18329) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:27:28,758] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:27:28,759] {logging_mixin.py:115} INFO - [2022-12-28 18:27:28,759] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:27:29,705] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:27:29,740] {logging_mixin.py:115} INFO - [2022-12-28 18:27:29,738] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:27:29,777] {logging_mixin.py:115} INFO - [2022-12-28 18:27:29,776] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:27:29,806] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.060 seconds
[2022-12-28 18:28:15,653] {processor.py:153} INFO - Started process (PID=18400) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:28:15,659] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:28:15,660] {logging_mixin.py:115} INFO - [2022-12-28 18:28:15,660] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:28:16,672] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:28:16,692] {logging_mixin.py:115} INFO - [2022-12-28 18:28:16,692] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:28:16,737] {logging_mixin.py:115} INFO - [2022-12-28 18:28:16,737] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-09-12T00:00:00+00:00, run_after=2022-09-19T00:00:00+00:00
[2022-12-28 18:28:16,769] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.130 seconds
[2022-12-28 18:31:03,651] {processor.py:153} INFO - Started process (PID=18706) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:31:03,668] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:31:03,670] {logging_mixin.py:115} INFO - [2022-12-28 18:31:03,669] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:31:04,443] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:31:04,460] {logging_mixin.py:115} INFO - [2022-12-28 18:31:04,460] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:31:04,490] {logging_mixin.py:115} INFO - [2022-12-28 18:31:04,490] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:31:04,520] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.892 seconds
[2022-12-28 18:31:34,875] {processor.py:153} INFO - Started process (PID=18750) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:31:34,876] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:31:34,877] {logging_mixin.py:115} INFO - [2022-12-28 18:31:34,877] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:31:35,545] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:31:35,580] {logging_mixin.py:115} INFO - [2022-12-28 18:31:35,579] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:31:35,612] {logging_mixin.py:115} INFO - [2022-12-28 18:31:35,611] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:31:35,647] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.778 seconds
[2022-12-28 18:32:10,605] {processor.py:153} INFO - Started process (PID=18829) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:32:10,610] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:32:10,611] {logging_mixin.py:115} INFO - [2022-12-28 18:32:10,611] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:32:11,286] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:32:11,305] {logging_mixin.py:115} INFO - [2022-12-28 18:32:11,304] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:32:11,329] {logging_mixin.py:115} INFO - [2022-12-28 18:32:11,328] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:32:11,362] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.762 seconds
[2022-12-28 18:32:48,448] {processor.py:153} INFO - Started process (PID=18895) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:32:48,453] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:32:48,454] {logging_mixin.py:115} INFO - [2022-12-28 18:32:48,454] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:32:49,079] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:32:49,096] {logging_mixin.py:115} INFO - [2022-12-28 18:32:49,096] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:32:49,119] {logging_mixin.py:115} INFO - [2022-12-28 18:32:49,119] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:32:49,145] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.701 seconds
[2022-12-28 18:33:28,299] {processor.py:153} INFO - Started process (PID=18972) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:33:28,300] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:33:28,301] {logging_mixin.py:115} INFO - [2022-12-28 18:33:28,301] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:33:28,959] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:33:28,978] {logging_mixin.py:115} INFO - [2022-12-28 18:33:28,978] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:33:29,002] {logging_mixin.py:115} INFO - [2022-12-28 18:33:29,001] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:33:29,031] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.744 seconds
[2022-12-28 18:34:00,377] {processor.py:153} INFO - Started process (PID=19038) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:34:00,378] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:34:00,380] {logging_mixin.py:115} INFO - [2022-12-28 18:34:00,380] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:34:01,080] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:34:01,096] {logging_mixin.py:115} INFO - [2022-12-28 18:34:01,096] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:34:01,119] {logging_mixin.py:115} INFO - [2022-12-28 18:34:01,119] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:34:01,199] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.830 seconds
[2022-12-28 18:34:31,577] {processor.py:153} INFO - Started process (PID=19090) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:34:31,715] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:34:31,721] {logging_mixin.py:115} INFO - [2022-12-28 18:34:31,721] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:34:32,449] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:34:32,467] {logging_mixin.py:115} INFO - [2022-12-28 18:34:32,467] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:34:32,491] {logging_mixin.py:115} INFO - [2022-12-28 18:34:32,491] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:34:32,516] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.945 seconds
[2022-12-28 18:35:03,887] {processor.py:153} INFO - Started process (PID=19153) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:35:03,889] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:35:03,891] {logging_mixin.py:115} INFO - [2022-12-28 18:35:03,891] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:35:04,510] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:35:04,540] {logging_mixin.py:115} INFO - [2022-12-28 18:35:04,540] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:35:04,565] {logging_mixin.py:115} INFO - [2022-12-28 18:35:04,565] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:35:04,626] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.744 seconds
[2022-12-28 18:35:38,788] {processor.py:153} INFO - Started process (PID=19214) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:35:38,794] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:35:38,795] {logging_mixin.py:115} INFO - [2022-12-28 18:35:38,795] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:35:39,882] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:35:40,034] {logging_mixin.py:115} INFO - [2022-12-28 18:35:40,034] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:35:40,071] {logging_mixin.py:115} INFO - [2022-12-28 18:35:40,071] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:35:40,120] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.342 seconds
[2022-12-28 18:36:13,912] {processor.py:153} INFO - Started process (PID=19272) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:36:13,914] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:36:13,919] {logging_mixin.py:115} INFO - [2022-12-28 18:36:13,919] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:36:14,840] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:36:14,861] {logging_mixin.py:115} INFO - [2022-12-28 18:36:14,860] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:36:14,907] {logging_mixin.py:115} INFO - [2022-12-28 18:36:14,906] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:36:14,951] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.058 seconds
[2022-12-28 18:36:49,744] {processor.py:153} INFO - Started process (PID=19338) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:36:49,746] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:36:49,748] {logging_mixin.py:115} INFO - [2022-12-28 18:36:49,747] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:36:50,634] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:36:50,691] {logging_mixin.py:115} INFO - [2022-12-28 18:36:50,691] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:36:50,702] {logging_mixin.py:115} INFO - [2022-12-28 18:36:50,702] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 18:36:50,736] {logging_mixin.py:115} INFO - [2022-12-28 18:36:50,736] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:36:50,818] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.080 seconds
[2022-12-28 18:38:25,727] {processor.py:153} INFO - Started process (PID=19492) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:38:25,746] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:38:25,747] {logging_mixin.py:115} INFO - [2022-12-28 18:38:25,747] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:38:26,794] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:38:26,819] {logging_mixin.py:115} INFO - [2022-12-28 18:38:26,819] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:38:26,866] {logging_mixin.py:115} INFO - [2022-12-28 18:38:26,866] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-10-03T00:00:00+00:00, run_after=2022-10-10T00:00:00+00:00
[2022-12-28 18:38:26,931] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.245 seconds
[2022-12-28 18:39:57,084] {processor.py:153} INFO - Started process (PID=19659) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:39:57,089] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:39:57,090] {logging_mixin.py:115} INFO - [2022-12-28 18:39:57,090] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:39:57,803] {logging_mixin.py:115} INFO - [2022-12-28 18:39:57,802] {dagbag.py:320} ERROR - Failed to import: /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/alxavier/Documents/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/alxavier/Documents/airflowalura/dags/dados_climaticos.py", line 46, in <module>
    strftime
NameError: name 'strftime' is not defined
[2022-12-28 18:39:57,804] {processor.py:653} WARNING - No viable dags retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:39:57,846] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.768 seconds
[2022-12-28 18:40:32,259] {processor.py:153} INFO - Started process (PID=19719) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:40:32,267] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:40:32,268] {logging_mixin.py:115} INFO - [2022-12-28 18:40:32,268] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:40:33,439] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:40:33,459] {logging_mixin.py:115} INFO - [2022-12-28 18:40:33,459] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:40:33,492] {logging_mixin.py:115} INFO - [2022-12-28 18:40:33,492] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:40:33,543] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.289 seconds
[2022-12-28 18:41:12,497] {processor.py:153} INFO - Started process (PID=19801) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:41:12,506] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:41:12,507] {logging_mixin.py:115} INFO - [2022-12-28 18:41:12,507] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:41:13,474] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:41:13,558] {logging_mixin.py:115} INFO - [2022-12-28 18:41:13,558] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:41:13,618] {logging_mixin.py:115} INFO - [2022-12-28 18:41:13,617] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 18:41:13,676] {logging_mixin.py:115} INFO - [2022-12-28 18:41:13,676] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:41:13,725] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.233 seconds
[2022-12-28 18:42:59,474] {processor.py:153} INFO - Started process (PID=19964) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:42:59,480] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:42:59,481] {logging_mixin.py:115} INFO - [2022-12-28 18:42:59,481] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:43:00,321] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:43:00,446] {logging_mixin.py:115} INFO - [2022-12-28 18:43:00,445] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:43:00,476] {logging_mixin.py:115} INFO - [2022-12-28 18:43:00,476] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 18:43:00,527] {logging_mixin.py:115} INFO - [2022-12-28 18:43:00,527] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:43:00,657] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.192 seconds
[2022-12-28 18:43:39,761] {processor.py:153} INFO - Started process (PID=20092) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:43:39,765] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:43:39,767] {logging_mixin.py:115} INFO - [2022-12-28 18:43:39,767] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:43:40,771] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:43:40,793] {logging_mixin.py:115} INFO - [2022-12-28 18:43:40,790] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:43:40,825] {logging_mixin.py:115} INFO - [2022-12-28 18:43:40,824] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:43:40,861] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.115 seconds
[2022-12-28 18:45:24,777] {processor.py:153} INFO - Started process (PID=20252) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:45:24,785] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:45:24,786] {logging_mixin.py:115} INFO - [2022-12-28 18:45:24,786] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:45:25,658] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:45:25,678] {logging_mixin.py:115} INFO - [2022-12-28 18:45:25,678] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:45:25,705] {logging_mixin.py:115} INFO - [2022-12-28 18:45:25,705] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-10-17T00:00:00+00:00, run_after=2022-10-24T00:00:00+00:00
[2022-12-28 18:45:25,761] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.990 seconds
[2022-12-28 18:47:08,488] {processor.py:153} INFO - Started process (PID=20470) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:47:08,502] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:47:08,503] {logging_mixin.py:115} INFO - [2022-12-28 18:47:08,503] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:47:09,139] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:47:09,208] {logging_mixin.py:115} INFO - [2022-12-28 18:47:09,208] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:47:09,220] {logging_mixin.py:115} INFO - [2022-12-28 18:47:09,220] {dag.py:2398} INFO - Creating ORM DAG for dados_climaticos
[2022-12-28 18:47:09,239] {logging_mixin.py:115} INFO - [2022-12-28 18:47:09,239] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:47:09,299] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.816 seconds
[2022-12-28 18:47:47,737] {processor.py:153} INFO - Started process (PID=20543) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:47:47,743] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:47:47,744] {logging_mixin.py:115} INFO - [2022-12-28 18:47:47,744] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:47:48,473] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:47:48,493] {logging_mixin.py:115} INFO - [2022-12-28 18:47:48,492] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:47:48,518] {logging_mixin.py:115} INFO - [2022-12-28 18:47:48,518] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:47:48,551] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.819 seconds
[2022-12-28 18:48:27,524] {processor.py:153} INFO - Started process (PID=20609) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:48:27,529] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:48:27,532] {logging_mixin.py:115} INFO - [2022-12-28 18:48:27,532] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:48:28,280] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:48:28,299] {logging_mixin.py:115} INFO - [2022-12-28 18:48:28,298] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:48:28,321] {logging_mixin.py:115} INFO - [2022-12-28 18:48:28,321] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:48:28,354] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.837 seconds
[2022-12-28 18:49:03,654] {processor.py:153} INFO - Started process (PID=20679) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:49:03,660] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:49:03,661] {logging_mixin.py:115} INFO - [2022-12-28 18:49:03,661] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:49:04,490] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:49:04,509] {logging_mixin.py:115} INFO - [2022-12-28 18:49:04,509] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:49:04,539] {logging_mixin.py:115} INFO - [2022-12-28 18:49:04,539] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:49:04,580] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.934 seconds
[2022-12-28 18:49:36,454] {processor.py:153} INFO - Started process (PID=20729) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:49:36,458] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:49:36,459] {logging_mixin.py:115} INFO - [2022-12-28 18:49:36,459] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:49:37,159] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:49:37,183] {logging_mixin.py:115} INFO - [2022-12-28 18:49:37,183] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:49:37,247] {logging_mixin.py:115} INFO - [2022-12-28 18:49:37,247] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:49:37,325] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.880 seconds
[2022-12-28 18:50:10,506] {processor.py:153} INFO - Started process (PID=20795) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:50:10,509] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:50:10,517] {logging_mixin.py:115} INFO - [2022-12-28 18:50:10,517] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:50:11,857] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:50:11,884] {logging_mixin.py:115} INFO - [2022-12-28 18:50:11,884] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:50:11,939] {logging_mixin.py:115} INFO - [2022-12-28 18:50:11,938] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:50:11,993] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.506 seconds
[2022-12-28 18:50:47,254] {processor.py:153} INFO - Started process (PID=20874) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:50:47,260] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:50:47,262] {logging_mixin.py:115} INFO - [2022-12-28 18:50:47,262] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:50:48,355] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:50:48,386] {logging_mixin.py:115} INFO - [2022-12-28 18:50:48,385] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:50:48,434] {logging_mixin.py:115} INFO - [2022-12-28 18:50:48,434] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-08-22T00:00:00+00:00, run_after=2022-08-29T00:00:00+00:00
[2022-12-28 18:50:48,464] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.221 seconds
[2022-12-28 18:52:50,154] {processor.py:153} INFO - Started process (PID=21081) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:52:50,161] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:52:50,162] {logging_mixin.py:115} INFO - [2022-12-28 18:52:50,162] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:52:50,911] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:52:50,931] {logging_mixin.py:115} INFO - [2022-12-28 18:52:50,931] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:52:50,962] {logging_mixin.py:115} INFO - [2022-12-28 18:52:50,962] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-10-17T00:00:00+00:00, run_after=2022-10-24T00:00:00+00:00
[2022-12-28 18:52:50,992] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.852 seconds
[2022-12-28 18:55:02,505] {processor.py:153} INFO - Started process (PID=21321) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:55:02,516] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:55:02,518] {logging_mixin.py:115} INFO - [2022-12-28 18:55:02,517] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:55:03,313] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:55:03,340] {logging_mixin.py:115} INFO - [2022-12-28 18:55:03,340] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:55:03,374] {logging_mixin.py:115} INFO - [2022-12-28 18:55:03,374] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:55:03,427] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 0.927 seconds
[2022-12-28 18:55:38,518] {processor.py:153} INFO - Started process (PID=21384) to work on /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:55:38,524] {processor.py:641} INFO - Processing file /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py for tasks to queue
[2022-12-28 18:55:38,525] {logging_mixin.py:115} INFO - [2022-12-28 18:55:38,525] {dagbag.py:507} INFO - Filling up the DagBag from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:55:39,394] {processor.py:651} INFO - DAG(s) dict_keys(['dados_climaticos']) retrieved from /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py
[2022-12-28 18:55:39,421] {logging_mixin.py:115} INFO - [2022-12-28 18:55:39,420] {dag.py:2379} INFO - Sync 1 DAGs
[2022-12-28 18:55:39,471] {logging_mixin.py:115} INFO - [2022-12-28 18:55:39,471] {dag.py:2927} INFO - Setting next_dagrun for dados_climaticos to 2022-12-26T00:00:00+00:00, run_after=2023-01-02T00:00:00+00:00
[2022-12-28 18:55:39,559] {processor.py:161} INFO - Processing /home/alxavier/Documents/airflowalura/dags/dados_climaticos.py took 1.056 seconds
